### PR TITLE
Gson and nexus 5 fix

### PR DESCRIPTION
**Pull request by NikolaHadzicNs**

### CODE REVIEW

**Comment by SlobodanPrijic** [comment]
@markokatic0001 review please

File: app/src/main/java/com/hyperether/citycamera/su/utils/LoginData.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    public User getUser() {
+        return user;
+    }
+
+    public void setUser(User user) {
+        this.user = user;
+    }
+
+    public class User {
```
Comment (SlobodanPrijic) [inline app/src/main/java/com/hyperether/citycamera/su/utils/LoginData.java]: @NikolaHadzicNs please set this class (and other sub classes) as separate file

File: app/src/main/java/com/hyperether/citycamera/su/activity/MainActivity.java [hunk 2]
```diff
@@ -131,13 +134,19 @@ public void onClick(View v) {
...
                         Gson gson = new Gson();
                         JsonElement element = gson.fromJson(response.toString(), JsonElement.class);
-                        JsonObject jsonObject = element.getAsJsonObject();
-                        String getUrl = jsonObject.get("url").getAsString();
-                        String fileId = jsonObject.get("fileId").getAsString();
-                        String url = getUrl.substring(getUrl.lastIndexOf("/") + 1);
-                        String baseUrl = getUrl.replace(url, "");
-
-                        uploadImage(baseUrl, url, file, mUploadToken, fileId);
+                        if (element.getAsJsonObject().has("error")) {
```
Comment (SlobodanPrijic) [inline app/src/main/java/com/hyperether/citycamera/su/activity/MainActivity.java]: @NikolaHadzicNs can you create one class that has both error and success response parts and do gson conversion at once?

**Comment by NikolaHadzicNs** [comment]
@SlobodanPrijic I made requested changes.

**Pull request merged**